releasenotes/notes/support_excitedstatessolver_bopes-2bbb5a282b1ff582.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+  - |
+    The solver of BOPESSampler now accepts solvers of type
```

Comment:
I think here BOPESSampler could a class ref like you have in the deprecations part so it forms a link the reader can click on.

Reply:
Done, thanks